manifests/init.pp

```diff
@@ -95,7 +101,9 @@
   String  $config_file            = $telegraf::params::config_file,
   String  $config_file_owner      = $telegraf::params::config_file_owner,
   String  $config_file_group      = $telegraf::params::config_file_group,
+  Filemode  $config_file_mode     = $telegraf::params::config_file_mode,
```

Comment:
The datatype is namespaced. Please update it like this:
```suggestion
  Stdlib::Filemode  $config_file_mode = $telegraf::params::config_file_mode,
```

Please ensure that the `=` are all aligned.